packages/now-cli/src/util/output/create-output.ts

```diff
@@ -120,8 +121,6 @@ export default function createOutput({ debug: debugEnabled = false } = {}) {
     return wait(message, delay);
   }
 
-  // This is pretty hacky, but since we control the version of Node.js
-  // being used because of `pkg` it's safe to do in this case.
```

Comment:
This is no longer "hacky"? 😄

Reply:
It's still hacky, but the part about `pkg` is no longer relevant 😛